### PR TITLE
faq update: windows LNK1112 error

### PR DESCRIPTION
**Pull request by plprofetes**
Quick update of FAQ: Compiling section with error I encountered myself when compiling ponyc.

### CODE REVIEW

File: content/faq/index.md [hunk 1]
```diff
@@ -242,6 +242,14 @@ For OpenSSL 1.1:
 ponyc -Dopenssl_1.1.0
 ```
 
+### On Windows I get "fatal error LNK1112: module machine type 'x86' conflicts with target machine type 'x64'"
+
+Only 64-bit Windows is supported.
+
+This error occurs when ponyc is compiled in Developer Command Prompt that is launched from 32-bit Visual Studio. 
+
+Make sure you're running pure cmd.exe/powershell.exe, that is not contaminated with VS's 32-bit env variables.
```
Comment (kulibali) [inline content/faq/index.md]: I would use more neutral language here.

```suggestion
Make sure you're running a cmd.exe/powershell.exe that does not include 32-bit VS environment variables.
```

File: content/faq/index.md [hunk 2]
```diff
@@ -242,6 +242,14 @@ For OpenSSL 1.1:
 ponyc -Dopenssl_1.1.0
 ```
 
+### On Windows I get "fatal error LNK1112: module machine type 'x86' conflicts with target machine type 'x64'"
+
+Only 64-bit Windows is supported.
+
+This error occurs when ponyc is compiled in Developer Command Prompt that is launched from 32-bit Visual Studio.
```
Comment (kulibali) [inline content/faq/index.md]: I would change the word order a bit.
```suggestion
This error occurs when ponyc is compiled in a 32-bit Visual Studio Developer Command Prompt.
```

Comment (plprofetes) [reply content/faq/index.md]: changed.

File: content/faq/index.md (hunk 1 above)
Comment (plprofetes) [reply content/faq/index.md]: changed.

**Comment by kulibali** [comment]
Awesome, thanks @plprofetes

**Pull request merged**